scaleway/resource_instance_security_group.go

```diff
@@ -170,33 +191,41 @@ func resourceScalewayInstanceSecurityGroupRead(d *schema.ResourceData, m interfa
 		}
 	}
 
-	_ = d.Set("inbound_rule", stateRules[instance.SecurityGroupRuleDirectionInbound])
-	_ = d.Set("outbound_rule", stateRules[instance.SecurityGroupRuleDirectionOutbound])
-	return nil
+	return stateRules, nil
 }
 
 func resourceScalewayInstanceSecurityGroupUpdate(d *schema.ResourceData, m interface{}) error {
 	meta := m.(*Meta)
 	instanceApi := instance.NewAPI(meta.scwClient)
 
-	zone, ID, err := parseZonedID(d.Id())
+	zone, securityGroupID, err := parseZonedID(d.Id())
```

Comment:
we kept the variable name `ID` across all resources, let's keep consistency here

Reply:
ok, but it makes `updateSecurityGroupeRules(d, zone, ID, instanceApi)`, l.238, harder to understand